test/evergreen/compatibility_test_for_releases.sh

```diff
@@ -63,21 +63,27 @@ build_branch()
     echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
     echo "Building branch: \"$1\""
     echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
+    
+    # Only clone the branch if the option is picked
+    if [ ! -d "build" ]; then
```

Comment:
Actually, should we check existence of `$1` or `$1/build` instead?

Reply:
Yes, have made that change.